pkg/models/shared_models.go

```diff
@@ -5,11 +5,11 @@ package models
 import "encoding/json"
 
 type IDModel struct {
-	ID int `json:"id,omitempty" tf:"id"`
```

Comment:
this is being used in many places, I hope this doesnt break anything. why do we need to remove omit_empty here ?

Reply:
updated